src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.match(/\S.*;/g);
+
+  const converted = styles.reduce((css, style) => {
+    const [property, value] = style
+      .replace(/\s*(?=:)|(?<=:)\s*(?=\S)|\s*;/g, '').split(':');
```

Comment:
🔥

Reply:
But also, don't forget about proper formatting
```suggestion
      .replace(/\s*(?=:)|(?<=:)\s*(?=\S)|\s*;/g, '')
      .split(':');
```